tests/NHSD.BuyingCatalogue.SolutionLists.API.UnitTests/SolutionListControllerTests.cs

```diff
@@ -62,6 +62,22 @@ public async Task ShouldListSolutions()
                 Times.Once);
         }
 
+        [Test]
+        public async Task ListAsync_HasSupplierId_ReturnsSolutionsByFilteredSupplierId()
+        {
+            var supplierId = "sup1";
+
+            var result = (await _solutionListController.ListAsync(supplierId).ConfigureAwait(false)).Result as ObjectResult;
+
+            result.StatusCode.Should().Be((int)HttpStatusCode.OK);
```

Comment:
We can use `StatusCodes.Status200OK` here, which will avoid the need to cast.

Reply:
👍